packages/theme/index.js

```diff
@@ -1,13 +1,70 @@
+const fs = require('fs');
+const path = require('path');
+const chalk = require('chalk');
+
+const capitalize = name => name.charAt(0).toUpperCase() + name.slice(1);
+const capEachWord = text => text.split(/[-_\s]/).map(capitalize).join('');
+const ilibVar = name => `ILIB_${name.toUpperCase().replace(/[-_\s]/g, '_')}_PATH`;
+
+const sourceFiles = (dir, action) => {
+	const list = fs.readdirSync(dir);
+	list.forEach(item => {
+		const itemPath = path.join(dir, item);
+		if (fs.statSync(itemPath).isDirectory()) {
+			sourceFiles(itemPath, action);
+		} else if (/\.(css|less|js|jsx|md)$/.test(item)) {
+			action(itemPath);
+		}
+	})
+};
+
+// Temporary fallback until ENYO-6133 is merged
+console.log = (log => (data, ...rest) =>
+	typeof data === 'undefined'
+		? log()
+		: typeof data === 'string'
+			? log(data.replace(/Enact app/, 'Enact theme'), ...rest)
+			: log.call(this, data, ...rest))(console.log);
+
 module.exports = {
+	type: 'theme',
+	setup: ({defaultGenerator, directory, name, skin = 'default-skin'}) => {
```

Comment:
If we rename 'my-skin' to 'default-skin' then our docs will be wrongly saying to update 'my-skin'.  Should we skip renaming skins if nothing is supplied?  How do you set the skin name?  Could we document how to do that if there's a way?

Reply:
We do update references within the readme.  Additionally, conceptually to me, `my-skin` conveys a 'demo-like' purpose, as an example, whereas `default-skin` as the output value in a new project (when an explicit skin name is omitted) fits better and requires no outright change my theme devs (when they only want the singular skin).

Updating both the template documentation and modifying the in-theme readme for the cli process is a separate ticket Dave is working on concurrently.

Reply:
:nice: Sounds good!